src/modules/vtol_att_control/vtol_att_control_main.h

```diff
@@ -162,6 +162,7 @@ class VtolAttitudeControl : public ModuleBase<VtolAttitudeControl>, public px4::
 
 	uORB::Subscription _action_request_sub{ORB_ID(action_request)};
 	uORB::Subscription _airspeed_validated_sub{ORB_ID(airspeed_validated)};			// airspeed subscription
+	float _calibrated_airspeed_m_s_smoothed{INFINITY};
```

Comment:
This seems like a strange location to define it

Reply:
Can problably be moved, chose this to be right next to the topic which it tracks. Anyway, naming should be improved.